main.cpp

```diff
@@ -376,26 +535,38 @@ struct Pickup
     std::string coverColor = "ivory";
     int stringNumber = 6;
 
-    void pickupStringVibration(bool pluckString = false);
-    void changeTheGuitarTone(int pickupSwitchPosition = 5);
-
-    //returns if pickup cover color matches the body color
-    bool matchTheBodyColor(SongWriter rockWriterA);
+    void pickupStringVibration( String fifthString, bool pluckString = false);
+    bool changeTheGuitarTone(int pickupSwitchPosition = 5, bool toneChanged = false);
+    void matchTheBodyColor(SongWriter rockWriterA);
 };
 
-/*
-Thing 10) Electric Guitar
-5 properties:
-    1) Neck
-    2) Body
-    3) Bridge
-    4) String
-    5) Pickup
-3 things it can do:
-    1) sell money
-    2) play rock music
-    3) collect dust
-*/
+void Pickup::pickupStringVibration(String fifthString, bool pluckString)
+{
+    if(pluckString)
+    {
+        fifthString.vibrate();
+    }
+}
+
+bool Pickup::changeTheGuitarTone(int pickupSwitchPosition, bool toneChanged)
+{
+    if(pickupSwitchPosition != 5)
+    {
+        toneChanged = true;
+    }
+    else
+    {
+        toneChanged = false;
+    }
```

Comment:
Don't pass in an argument if you're not going to use it.
just declare `toneChanged` as a local variable and remove it as a function parameter.
- [ ] revise